.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+If you added a new package make sure to check for these items:
+- In the `README.md` file in the project root add a new connection in `graph TD` code block section.
+This adds a connection to a graph created by [Mermaid](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/).
+- For your package also create a `README.md` file.
+- Run `npm install` and `npx eslint --print-config .eslintrc`, and check if the config is correct,
+especially if you are extending an existing package or adding an external config as a dependency (e.g. the
+external config might override your rules).
```

Comment:
Don't manually split into multiple lines.

Reply:
Done
https://github.com/infinum/js-linters/pull/66/commits/fdae3353fa07f3ea63f55ec2715880bde1c3ef90

This is currently my last PR in any of our repos, so I will remember this for the future :D